src/psyclone/psyGen.py

```diff
@@ -2766,6 +2767,8 @@ def apply(self, node, options=None, **kwargs):
         '''
         # TODO 2668: options are now deprecated:
         if options is not None:
+            logger = logging.getLogger(__name__)
```

Comment:
What is the difference of having this line or using `logging.warning(message)` directly?